examples/hello_worlds.rb

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ruby
+
+require 'vedeu'
+
+# An example application to demonstrate 'Hello World' and a minimum of
+# interactivity.  It uses the DSL-style of vedeu (not the full-flegded
+# MVC-approach).
+#
+# If you have cloned this repository from GitHub, you can run this example:
+#
+#     bundle exec ./examples/hello_worlds.rb
+#
+class HelloWorldsApp
+
+  Vedeu.bind(:_initialize_) { Vedeu.trigger(:_refresh_) }
+
+  Vedeu.configure do
+    # Empty configure block is needed.
+  end
+
+  Vedeu.interface :hello do
+    # Define all of the interface in one place.
+    background '#000000'
+    foreground '#00ff00'
+    geometry do
+      centred!
+      height   5
+      width    24
+    end
+    Vedeu.views do
```

Comment:
`Vedeu.views` are not really part of the interface DSL, but if this is working for you- then I can't see it as a problem!